gpflow/likelihoods/robustmax.py

```diff
@@ -22,7 +23,8 @@ class RobustMax(tf.Module):
     def __init__(self, num_classes, epsilon=1e-3, **kwargs):
         super().__init__(**kwargs)
         transform = tfp.bijectors.Sigmoid()
-        prior = tfp.distributions.Beta(0.2, 5.)
+        fdtype = lambda x: np.array(x, dtype=default_float())
```

Comment:
what does the `f` stand for?
if this is something needed throughout the code base, would it be worthwhile putting it into `gpflow.utilities` or so ?